tests/test_costs.py

```diff
@@ -4,7 +4,7 @@
 from webservices.rest import api
 from webservices.common.models import CommunicationCost, Electioneering
 from webservices.schemas import CommunicationCostSchema, ElectioneeringSchema
-from webservices.resources.costs import CommunicationCostView, ElectioneeringView
+from webservices.resources.costs import CommunicationCostView, ElectioneeringView, CommunicationCostViewOffset
 
 
 class TestCommunicationCost(ApiBaseTest):
```

Comment:
after 30 days, we will clean this file. 
Please add comment here to show which endpoint this test case for. Thanks

Reply:
@fec-jli I made this part of the issue to add the new endpoint already: https://github.com/fecgov/openFEC/issues/4546